### PR TITLE
[changelog] Latest changes backported to 8.12 branch.

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -442,6 +442,12 @@ Tactics
   by Pierre Courtieu,
   fixes `#11761 <https://github.com/coq/coq/issues/11761>`_,
   reported by Lasse Blaauwbroek).
+- **Fixed:**
+  typeclasses eauto (and discriminated hint bases) now correctly
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:446]:
```suggestion
  :tacn:`typeclasses eauto` (and discriminated hint bases) now correctly
```

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -483,6 +489,17 @@ Tactic language
...
   <https://github.com/coq/coq/issues/12196>`_, `#12197
   <https://github.com/coq/coq/pull/12197>`_, by Jason Gross).
+- **Fixed:**
+  Excluding occurrences was causing an anomaly in tactics
+  (e.g., :g:`pattern _ at L` where :g:`L` is :g:`-2`).
+  (`#12541 <https://github.com/coq/coq/pull/12541>`_,
+  fixes `#12228 <https://github.com/coq/coq/issues/12228>`_,
+  by Pierre Roux).
+- **Fixed:**
+  Fix the parsing of multi-parameters Ltac2 types
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:499]:
```suggestion
- **Fixed:**
  Parsing of multi-parameters Ltac2 types
```

**Review by Zimmi48 — CHANGES_REQUESTED** [review]
You should create a new section "Changes in 8.12.0" instead of adding the changelog entries to the "Changes in 8.12+beta1" section.

**Comment by ejgallego** [comment]
Oh indeed, fixed; thanks!

**Review by Zimmi48 — APPROVED** [review]
Let's indeed wait for #12678 to be in before updating and merging this.

**Comment by ejgallego** [comment]
This is good to go I think.

**Comment by Zimmi48** [comment]
I just noticed that you have inserted the new section on 8.12.0 before the one on 8.12+beta1. In previous releases, we did the converse (the most important changes being in the first beta, they are listed first, despite the overall page being listed in reverse chronological order).

**Comment by ejgallego** [comment]
> I just noticed that you have inserted the new section on 8.12.0 before the one on 8.12+beta1. In previous releases, we did the converse (the most important changes being in the first beta, they are listed first, despite the overall page being listed in reverse chronological order).

Oh; I see. Should I reverse? I guess I'm used to most changelogs going in chronological order.

**Comment by Zimmi48** [comment]
I guess so. If we want to change this, we should discuss it first and change it everywhere.

**Comment by ejgallego** [comment]
Ok done.

**Pull request merged**